salt/syspaths.py

```diff
@@ -136,6 +136,10 @@
 if SPM_REACTOR_PATH is None:
     SPM_REACTOR_PATH = os.path.join(SRV_ROOT_DIR, 'spm', 'reactor')
 
+HOME_DIR = __generated_syspaths.SPM_REACTOR_PATH
```

Comment:
I'm unclear on why `SPM_REACTOR_PATH` was chosen here. Could you please clarify?